### PR TITLE
Migrating to the use of in-toto/go-witness module

### PR DESCRIPTION
**Pull request by ChaosInTheCRD**
to be merged after https://github.com/in-toto/witness/pull/328

### CODE REVIEW

File: cmd/verify_test.go
Comment (ChaosInTheCRD) [inline cmd/verify_test.go:317]: We change this now that witness follows the new go-witness module

**Comment by ChaosInTheCRD** [comment]
This branch uses a **non release** version of `go-witness` - wait for a release of `go-witness` before merging this PR

**Review by jkjell — APPROVED** [review]

**Pull request merged**
